docs/docs/workflows/troubleshooting.md

```diff
@@ -106,3 +106,19 @@ Debugging via a shell on a node is [directly supported by Kubernetes](https://ku
     ```sh
     kubectl delete pod node-debugger-constell-worker-xksa0-000000-bjthj
     ```
+
+# Changing measurements
+
+There might be situations in which you have a running Constellation cluster, but want to change the expected measurements.
+For example because the cloud provider updated the firmware.
+You can achieve this by using the `upgrade apply` command:
```

Comment:
```suggestion
## Nodes fail to join with error `untrusted PCR value`

This error indicates that a node's [attestation statement](../architecture/attestation.md) contains measurements that don't match the trusted values expected by the [JoinService](../architecture/microservices.md#joinservice).
This may for example happen if the cloud provider updates the VM's firmware such that it influences the [runtime measurements](../architecture/attestation.md#runtime-measurements) in an unforeseen way.
You can change the expected measurements to resolve the failure.

:::caution

Attestation and trusted measurements are crucial for the security of your cluster.
Be extra careful when manually changing these settings.

When in doubt, check if the encountered [issue is known](https://github.com/edgelesssys/constellation/issues?q=is%3Aopen+is%3Aissue+label%3A%22known+issue%22) or [contact support](https://github.com/edgelesssys/constellation#support).

:::

You can use the `upgrade apply` command to change measurements of a running cluster:
```
Suggest to emphasize the issue instead of the action. We currently have both types of sections on this page, but to me this fits better for this item. If you agree, I'll rearrange this page a bit after this PR is merged.
I also added a bit more context and motivation. Do you agree with that?

Reply:
I do.